girder/LICENSE

```diff
@@ -0,0 +1,15 @@
+Apache Software License 2.0
+
+Copyright (c) 2019, Patrick Avery
```

Comment:
This should be Kitware Inc. :smile:

Reply:
Also I think we want 3 clause BSD

Reply:
Haha :laughing: . Yeah, I'll change that to Kitware, and 3-clause BSD.